perfaware/sim86/shared/Sim8086.cpp

```diff
@@ -8,6 +8,8 @@
 #pragma comment (lib, "sim86_shared_debug.lib")
 #define ArrayCount(Array) (sizeof(Array) / sizeof((Array)[0]))
 
+u8* Memory;
+static constexpr size_t Megabyte = 1024 * 1024;
 static constexpr int RegisterCount = 15;
 static constexpr int IPRegister = 13;
 static constexpr int CXRegister = 3;
```

Comment:
About all these constants, your naming convention doesn't have anything to distinguish them from non-constant values. This is something that is nice to have, so you may want to think about a way to show that they're constants. Common ways to do that are to use uppercase names, or other schemes like that.

So when people read your code, and suddenly there's a new name that appears out of nowhere, that wasn't declared inside this function, etc, people will know "oh ok it's a constant, not something that I missed here".

Reply:
Good point. Constants are now shouted across the program. 
8ff251f7ee79bf0b6365f7a89c0361cd9c046cb9

Reply:
THANKS. Just a note, but with this style words are usually separated by underscore for readability.

Reply:
Much better indeed
827163329bb1769d988192652596daef257b732a